### PR TITLE
Feature FBFW-117/118 Retreats Phase

### PR DESCRIPTION
**Pull request by jasonekstrom**
Retreats phase: dislodge, retreat and disband functionality

### CODE REVIEW

**Review by sariacodazen — APPROVED** [review]
LGTM

File: beta-src/src/state/game/game-api-slice.ts [hunk 1]
```diff
@@ -167,13 +168,21 @@ const setCommand = (
...
+      command: phase === "Retreats" ? "DISLODGED" : "NONE",
     };
     setCommand(state, command, "unitCommands", unitID);
+    command = {
+      command: "REMOVE_ARROW",
+      data: {
+        orderID,
+      },
+    };
+    setCommand(state, command, "mapCommands", "all");
```
Comment (bryancodazen) [inline beta-src/src/state/game/game-api-slice.ts:185]: You shouldn't need this code, and it is causing errors on the diplomacy phase.

File: beta-src/src/utils/map/getOrdersMeta.ts [hunk 2]
```diff
@@ -38,9 +39,17 @@ export default function getOrdersMeta(data, phase): Props {
...
+          orderUnit = new UnitClass({
+            ...units[unitID],
+            Territory: territories[units[unitID].terrID],
+          });
+          newBoard.units.push(orderUnit);
+        }
         if (orderUnit) {
-          newOrders.push(new OrderClass(newBoard, o, orderUnit));
+          const tempOrder = new OrderClass(newBoard, o, orderUnit);
+          newOrders.push(tempOrder);
```
Comment (bryancodazen) [inline beta-src/src/utils/map/getOrdersMeta.ts:52]: We should remove these unnecessary changes.

**Review by bryancodazen — CHANGES_REQUESTED** [review]
When I tried building the code ESLint/Prettier threw an error, can you please make sure formatting is correct so we can build when pulling down.
